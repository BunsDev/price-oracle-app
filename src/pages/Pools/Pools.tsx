import Dashboard from './Dashboard/Dashboard';
import PoolList from './PoolList/PoolList';

function Pools() {
  return (
    <>
      <Dashboard />
      <PoolList />
    </>
  );
}

export default Pools;
