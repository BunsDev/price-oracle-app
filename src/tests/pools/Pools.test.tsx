import Pools from '~/pages/Pools';
import { render } from '~/tests';

describe('Pool Page', () => {
  it('Should render Pools Page', () => {
    const { asFragment, getByText } = render(<Pools />);

    expect(getByText(/dashboard/i)).toBeInTheDocument();
    expect(asFragment()).toMatchSnapshot();
  });
});
