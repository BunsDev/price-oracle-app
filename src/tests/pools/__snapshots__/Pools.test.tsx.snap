// Vitest Snapshot v1

exports[`Pool Page > Should render Pools Page 1`] = `
<DocumentFragment>
  <div
    class="sc-iOeugr caIFBI"
  >
    <div
      class="sc-iBYQkv sc-jrcTuL blHKRc epIsAC sc-ipEyDJ sc-iAEawV hhOCqg dINGjr"
      color="rgba(255,255,255, 0.87)"
    >
      <div
        class="sc-eeMvmM jcHRro"
      >
        <div
          class="sc-iBYQkv sc-jrcTuL hNdzWR epIsAC"
          color="rgba(255,255,255, 0.87)"
        >
          Dashboard
        </div>
      </div>
      <div
        class="sc-cUEOzv"
      >
        <div
          class="sc-iBYQkv sc-jrcTuL cWFtuj epIsAC"
          color="rgba(255,255,255, 0.87)"
        >
          Total ETH locked
        </div>
      </div>
      <div
        class="sc-cUEOzv"
      >
        <div
          class="sc-iBYQkv sc-jrcTuL cWFtuj epIsAC"
          color="rgba(255,255,255, 0.87)"
        >
          Claimable rewards
        </div>
      </div>
      <div
        class="sc-jOiSOi iuJImF"
      >
        <div
          class="sc-fLcnxK"
        >
          <i
            class="sc-hBxehG sc-bBABsx ccFtCM jFMzlb picon-loader"
            name="loader"
          />
        </div>
      </div>
      <div
        class="sc-jOiSOi iuJImF"
      >
        <div
          class="sc-fLcnxK"
        >
          <i
            class="sc-hBxehG sc-bBABsx ccFtCM jFMzlb picon-loader"
            name="loader"
          />
        </div>
      </div>
    </div>
    <div
      class="sc-iBYQkv sc-jrcTuL blHKRc epIsAC sc-ipEyDJ sc-dPWrhe hhOCqg bkVTXd"
      color="rgba(255,255,255, 0.87)"
    >
      <div
        class="sc-iBYQkv sc-jrcTuL hNdzWR epIsAC sc-bCfvAP"
        color="rgba(255,255,255, 0.87)"
      >
        Pools
      </div>
      <div
        class="sc-ilhmMj fASgVG"
      >
        <i
          class="sc-hBxehG sc-jIRcFI ccFtCM fJYkmv picon-search"
          name="search"
        />
        <input
          aria-label="Search"
          class="sc-hHTYSt sc-dmctIk sc-hhOBVt fvoNMM jgGokt kaBZVh"
        />
      </div>
      <div
        class="sc-jfTVlA dGXjYf"
      >
        <div
          class="sc-fLcnxK"
        >
          <i
            class="sc-hBxehG sc-bBABsx ccFtCM jFMzlb picon-loader"
            name="loader"
          />
        </div>
      </div>
    </div>
  </div>
</DocumentFragment>
`;
